ghost/core/core/server/data/migrations/versions/5.83/2024-05-09-09-28-44-add-status-index-for-posts.js

```diff
@@ -0,0 +1,13 @@
+// For information on writing migrations, see https://www.notion.so/ghost/Database-migrations-eb5b78c435d741d2b34a582d57c24253
+
+const {createNonTransactionalMigration} = require('../../utils');
+const {addIndex, dropIndex} = require('../../../schema/commands');
+
+module.exports = createNonTransactionalMigration(
```

Comment:
You can simplify this by using `createAddIndexMigration`

https://github.com/TryGhost/Ghost/blob/842290cbef092ea438c5ea737342c3ecad90445f/ghost/core/core/server/data/migrations/versions/5.72/2023-10-31-11-06-00-members-created-attribution-id-index.js#L2